JunQueue.java

```diff
@@ -1,7 +1,10 @@
+import java.util.Arrays;
+
 public class JunQueue {
     private String[] queue;
     private int trackQueuePosition;
 
+
     public JunQueue() {

```

Comment:
Overall great job here too, style stuff with import statement but everything else is completely correct, once again readable and simple.